JniBridge.bee.cs

```diff
@@ -466,7 +474,7 @@ static NativeProgram SetupTestProgramWindows(ToolChain toolchain, NativeProgram
 
         var destDir = new NPath("build").Combine(Platform.Windows);
         var config = new NativeProgramConfiguration(codegen, toolchain, false);
-        var target = np.SetupSpecificConfiguration(config, config.ToolChain.ExecutableFormat).DeployTo(destDir);
+        var target = np.SetupSpecificConfiguration(config, config.ToolChain.ExecutableFormat).DeployTo(destDir);
```

Comment:
I have no idea what  has changed in this line and the line below, but automatic code formatter changes something if this file is saved even without any actual changes.